app/components/icons/ArrowUp.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function ArrowUp() {
+  return (
+    <div>
```

Comment:
The SVGs don't necessarily need the `div` wrapper, the `svg` tag alone is enough :)